stac_fastapi/extensions/stac_fastapi/extensions/core/collection_search/request.py

```diff
@@ -0,0 +1,27 @@
+"""Request models for the Collection-Search extension."""
+
+from typing import Optional
+
+import attr
+from fastapi import Query
+from stac_pydantic.shared import BBox
+from typing_extensions import Annotated
+
+from stac_fastapi.types.rfc3339 import DateTimeType
+from stac_fastapi.types.search import APIRequest, _bbox_converter, _datetime_converter
+
+
+@attr.s
+class CollectionSearchExtensionGetRequest(APIRequest):
+    """Basics additional Collection-Search parameters for the GET request."""
+
+    bbox: Optional[BBox] = attr.ib(default=None, converter=_bbox_converter)
+    datetime: Optional[DateTimeType] = attr.ib(
+        default=None, converter=_datetime_converter
+    )
+    limit: Annotated[
+        Optional[int],
+        Query(
+            description="Limits the number of results that are included in each page of the response."  # noqa: E501
+        ),
+    ] = attr.ib(default=None)
```

Comment:
should we put a default to the limit ?

Reply:
The intention is to behave consistent with Item Search. What does Item Search put?

Reply:
Item Search set `10` as default! https://github.com/stac-utils/stac-fastapi/blob/fe4d0dfb9812ec624b5af8bb10e32621b48e090a/stac_fastapi/types/stac_fastapi/types/search.py#L176-L181